lib/subordinateio.c

```diff
@@ -685,6 +686,24 @@ uid_t sub_uid_find_free_range(uid_t min, uid_t max, unsigned long count)
 	return start == ULONG_MAX ? (uid_t) -1 : start;
 }
 
+bool want_subuid_file(void)
```

Comment:
Please include in the commit message the reason for the rename.

Reply:
Mostly because we might want_subuids even when we don't want_subuid_file, but it's just my 5¢

Reply:
Hmmm, sounds reasonable, I guess.